conf/modules.config

```
@@ -220,15 +234,29 @@ process {
         ]
     }
 
+    // ------------------------------------
+    // Shiny app
+    // ------------------------------------
+    withName: 'PREP_SHINY' {
```

I would call the process `PREPARE_SHINY` instead
```suggestion
    withName: 'PREPARE_SHINY' {
```